### ISSUE TITLE
Default colors for Normal and Low Vision modes

### ISSUE THREAD

**Issue by shepazu** [fizzstudio/ParaCharts#577]
Our default colors are not currently as accessible as they should be. 

Eventually, we'll need to get a color designer with accessibility experience to help us craft color palettes that are attractive, complementary, and have sufficient contrast with the background, with text, and with each other, including custom palettes for people with each type of color blindness (CVD).

But for now, here are 2 sets of colors that are at least WCAG AA conformant for color contrast with white "normal text", and WCAG AAA for "large text". For reference, large text is `18 point` / `1.5 em` or `14 point bold` / `1.2 em bold`.

The order of the colors is also important: the first several colors should be nameable colors, since there's evidence that that decreases cognitive load, and makes them easier to talk about.

Note that this assumes a white background and white text; in dark/high-contast/low-vision modes, the background will be black or dark gray by default, so we should test against that. We should also come up with other palettes for other background colors, eventually.

## Normal default

These are the first 5 colors for the default palette, more muted so they don't clash with each other. They are distinguishable from each other for protanopia, deuteranopia, and tritanopia.

```
--color-blue-1: hsl(227, 26%, 52%);
--color-red-1: hsl(13, 58%, 35%);
--color-green-1: hsl(109, 93%, 24%);
--color-purple-1: hsl(300, 50%, 50%);
--color-blue-green-1: hsl(180, 100%, 25%);
```

Let's review the subsequent colors, too, after these first five, to make sure there's sufficient contrast.

## Low vision default

These are the first 5 colors for the default palette for low-vision mode. They are more primary and vibrant, since we've heard from some low-vision folks that they are easier to distinguish than the more muted palette:

```
--color-blue-2: hsl(240, 100%, 60%);
--color-red-2: hsl(0, 100%, 50%);
--color-green-2: hsl(120, 100%, 33%);
--color-magenta-1: hsl(300, 77%, 54%);
--color-brown-1: hsl(39, 100%, 31%);
```
